contrib/hooks/clang-format-pre-commit.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Copyright (c) 2021 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# A pre-commit hook that applies clang-format-9 to all the changes
+# (staged and unstaged).
+# After the formatting has been applied, it will ask you which changes should
+
+echo "Formatting all changed files now..."
+git clang-format-9 -f
```

Comment:
I have a feeling this does not work in windows git shell

Reply:
Yes, this would be a separate PR. However, we want to fix the clang-format version. As we decided once, to not fix the clang-format rules for version specific stuff, we would end up in a difference between the CI and the local formatting.